### DISCUSSION TITLE
Include figma block automatically on docs page

### DISCUSSION THREAD

**Discussion by manuelmeister** [storybookjs/addon-designs#130] [Q&A]
How can I show the figma block automatically on the docs page if the design param is set on the default export or the story?
I would want it collapsed so it doesn't load until someone cares.

P.S. Such a cool addon! :)

**Comment by pocka** [comment]
You can modify the default Docs Page by [remixing](https://storybook.js.org/docs/react/writing-docs/docs-page#remixing-docspage-using-doc-blocks).

```ts
// I haven't tested this code, might not work
import { Design } from "storybook-addon-designs/blocks";

// ...
parameters: {
  docs: {
    page: () => (
      <>
        <Title />
        <Subtitle />
        <Description />
        <Primary />
        <ArgsTable story={PRIMARY_STORY} />
        <Design storyId={PRIMARY_STORY} />
        <Stories />
      </>
    ),
  },
},
```
